### PR TITLE
samg55-oled1: initial commit

### PR DESCRIPTION
**Pull request by jakubruczynski**

### CODE REVIEW

File: source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+[OLED1 Xplained Pro](http://www.atmel.com/Images/Atmel-42077-OLED1-Xplained-Pro_User-Guide.pdf)
+
+[SSD1306 controller API](http://asf.atmel.com/docs/latest/samg/html/group__ssd1306__oled__controller__group.html)
+
+Summary
+-------
+
+As you can see, starting with `Atmel SAMG55 Xplained Pro` can be easy. I hope
+that providied information are easy to read, and useful. If they are not,
```
Comment (pietrushnic) [inline source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown]: @jakubruczynski providied?

File: source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+Before connecting, remember to change baudrate to `115200`.  Now you are all
+set up, and ready to code.
+
+Sources
+-------
+
+SAMG55 Xplained Pro documentation
+ 
+[Data Gateway Interface](http://www.atmel.com/Images/Atmel-32223-Data-Gateway-Interface_UserGuide.pdf)
+
```
Comment (pietrushnic) [inline source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown:132]: @jakubruczynski no need of empty line here - looks ugly after rendering. Please also add bullets for those links.

**Comment by pietrushnic** [comment]
@jakubruczynski it would be great to have picture of displayed `Hello World` right before summary.

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown (hunk 2 above)
Comment (jakubruczynski) [reply source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown:132]: @pietrushnic corrected, indeed looks a lot better

File: source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown (hunk 1 above)
Comment (jakubruczynski) [inline source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown]: @pietrushnic what is wrong with that word?

Comment (jakubruczynski) [reply source/_posts/2016-11-7-samg55-oled1-xplained-pro.markdown]: @pietrushnic corrected

**Comment by jakubruczynski** [comment]
@pietrushnic i added the photo, but I'm not sure if this is what should be there, it does not say "hello world"

**Pull request merged**